style.css

```diff
@@ -309,6 +307,67 @@ form div .view-more {
   flex-direction: column;
 }
 
+.work-modal {
+  background-color: #fff;
+  border-radius: 16px;
+  margin: 4%;
+  padding: 10px;
+}
+
+.work-modal-head {
+  display: flex;
+  justify-content: space-between;
+}
+
+.work-modal-head > .title {
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.close-modal {
+  font-size: 35px;
+  color: #67798e;
+  margin-top: -10px;
+  cursor: pointer;
+}
+
+.work-modal-hr {
+  color: #ebecf0;
+}
+
+.work-modal-img {
+  width: 100%;
+  height: auto;
+}
+
+#techs {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  justify-content: center;
+}
+
+#techs-item {
+  margin-top: 10px;
+}
+
+#btn-link {
+  text-decoration: none;
+}
+
+.btn-icon {
+  margin-left: 7px;
+  margin-bottom: -8px;
+}
+
```

Comment:
- [ ] You made great work so far but your design doesn't reflect the Figma design
as required for this project The popup should follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins), I can see that your popup is not `fixed` for both desktop and mobile version, when scrolling down and up it really looks ugly because is not fixed anymore :smile: , kindly fix this for your design to match the original design :wink: 

Check on this video and see how your popup is not fixed :point_down: 

https://user-images.githubusercontent.com/88894525/223502328-b5822105-80fb-4edf-8c27-f97ae5c24bc5.mp4

Figma Design 
![Screenshot from 2023-03-07 19-15-20](https://user-images.githubusercontent.com/88894525/223502441-aa0ca6f3-df18-4554-b0cf-595a74c5d09a.png)

Reply:
done